SoftLayer/CLI/virt/capacity/create.py

```diff
@@ -0,0 +1,57 @@
+"""Create a Reserved Capacity instance."""
+
+import click
+
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.managers.vs_capacity import CapacityManager as CapacityManager
+
+
+@click.command(epilog=click.style("""WARNING: Reserved Capacity is on a yearly contract"""
+                                  """ and not cancelable until the contract is expired.""", fg='red'))
+@click.option('--name', '-n', required=True, prompt=True,
+              help="Name for your new reserved capacity")
+@click.option('--datacenter', '-d', required=True, prompt=True,
+              help="Datacenter shortname")
+@click.option('--backend_router_id', '-b', required=True, prompt=True,
+              help="backendRouterId, create-options has a list of valid ids to use.")
+@click.option('--capacity', '-c', required=True, prompt=True,
+              help="Capacity keyname (C1_2X2_1_YEAR_TERM for example).")
+@click.option('--quantity', '-q', required=True, prompt=True,
```

Comment:
quantity in this case means how many instances a capacity group should have, not how many groups to order. Perhaps I should rename that to be a bit more clear to users since there is also capacity.